net/ipv4/tcp_output.c

```diff
@@ -2703,17 +2703,13 @@ static bool tcp_write_xmit(struct sock *sk, unsigned int mss_now, int nonagle,
 							  max_segs),
 						    nonagle);
 #ifdef CONFIG_SECURITY_TEMPESTA
-/*
- * Maximum number of bytes, that the hpack dynamic
- * table size can occupy.
- */
-#define HPACK_TBL_BYTES_OCCUPIED_MAX 3
-		if (sk->sk_write_xmit
+		if (sk->sk_prepare_xmit
 		    && (tempesta_skb_get_cb_val(skb, TEMPESTA_TLS_SKB_TYPE_OFF,
 						TEMPESTA_TLS_SKB_TYPE_MAX)
 			& TEMPESTA_TLS_SKB_TYPE_MAX)) {
```

Comment:
Such multi-line conditions are hard to read, so could you place place `{` on a new line